pkg/column-mapping/column.go

```diff
@@ -507,13 +507,13 @@ func computePartitionID(schema, table string, rule *Rule) (instanceID int64, sch
 
 func computeID(name string, prefix string, bitSize int, shiftCount uint) (int64, error) {
 	if len(prefix) >= len(name) || prefix != name[:len(prefix)] {
-		return 0, errors.NotValidf("%s is not prefix of %s", prefix, name)
+		return 0, errors.NotValidf("%s is not the prefix of %s", prefix, name)
 	}
 
 	idStr := name[len(prefix):]
 	id, err := strconv.ParseUint(idStr, 10, bitSize)
 	if err != nil {
-		return 0, errors.NotValidf("suffix of %s is not int64", idStr)
+		return 0, errors.NotValidf("suffix of %s's data type is not int64", idStr)
```

Comment:
we should focus on `idStr`, can we change the statement not followed by `'s`?

Reply:
like this :"suffix of the idStr, can't be converted to int64" ?

Reply:
`suffix of the idStr can't be converted to int64`?